pkg/storage/filepath/jsonfile_test.go

```diff
@@ -106,12 +107,17 @@ type fixture struct {
 	t       *testing.T
 	dir     string
 	storage rest.StandardStorage
+	ctx     context.Context
+	cancel  context.CancelFunc
 }
 
 func newFixture(t *testing.T, fs filepath.FS) *fixture {
 	dir, err := ioutil.TempDir("", strings.Replace(t.Name(), "/", "_", -1))
 	require.NoError(t, err)
 
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Minor, but I always like to reduce the likelihood the test leaks something:
```suggestion
	ctx, cancel := context.WithCancel(context.Background())
	t.Cleanup(cancel)
```

Reply:
Yeah, I would normally do that too. This particular fixture has a teardown method that is deferred from the test function, so I put the cancel execution in the method.

Reply:
Ahhh missed that!